web/cobrands/fixmystreet.com/_colours.scss

```diff
@@ -16,6 +16,9 @@ $nav_background_colour: #222;
 $nav_colour: #fff;
 $nav_hover_background_colour: #444;
 
+$link-color: #005A9E;
+$link-hover-color: #1573b9;
```

Comment:
If these are improving accessibility, should we not change the default sass/_base.scss versions of these instead?